src/sentry/api/endpoints/integrations/sentry_apps/installation/index.py

```diff
@@ -53,6 +56,29 @@ def post(self, request: Request, organization) -> Response:
         if not app.status == SentryAppStatus.PUBLISHED and app.owner_id != organization.id:
             return Response(status=404)
 
+        # feature check
+        app_features = IntegrationFeature.objects.filter(
+            target_id=app.id, target_type=IntegrationTypes.SENTRY_APP.value
+        )
+
+        is_feature_enabled = {}
+        for feature in app_features:
+            feature_flag_name = "organizations:%s" % feature.feature_str()
+            is_feature_enabled[feature_flag_name] = True
+            if feature_flag_name in settings.SENTRY_FEATURES:
```

Comment:
I saw you used this in the previous pr, but it's actually not totally correct. Features can be registered dynamically too, and we do that in some places in getsentry.

Instead, Just use `features.has` and catch `FeatureNotRegistered` to see if the feature does not exist.